links-slackbot/src/test/java/juja/microservices/links/slackbot/util/SlackTextHandlerTest.java

```diff
@@ -1,7 +1,46 @@
 package juja.microservices.links.slackbot.util;
 
+import org.hamcrest.collection.IsIterableContainingInAnyOrder;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+
 /**
- * @author Ivan Shapovalov
+ * @author Ben Novikov
  */
 public class SlackTextHandlerTest {
+    private final SlackTextHandler slackTextHandler = new SlackTextHandler();
+
+    @Test
+    public void getURLsFromText_getSet_returnsExpectedSet() {
+        String text =
+                "Ссылка <> <http://google.com?parameter=value&also=another|google.com> " +
+                "Ссылка <1> http://google.com " +
+                "Ссылка <2 <yandex.ru/page?parameter=value&also=another> " +
+                "Ссылка 3> https://google.info " +
+                "Ссылка 4 <<https://mail.ru> " +
+                "Ссылка 5 <<> google.ya " +
+                "Ссылка 6 <http://yahoo.us|yahoo.us> " +
+                "Ссылка 7 <mail@microsoft.com " +
+                "Ссылка 8 <mailto:mail.mail@post.by|mail@post.by> " +
+                "Ссылка 9 ftp://ya.google.au> " +
+                "Ссылка 9 <ftp://ya.google.mail.ya.au> " +
+                "Ссылка 10 <ftp://name:pass@warehouse.com> " ;
+        List<String> expectedList = new ArrayList<>(Arrays.asList(
+                "http://google.com?parameter=value&also=another",
+                "https://mail.ru",
+                "http://yahoo.us",
+                "ftp://ya.google.mail.ya.au",
+                "ftp://name:pass@warehouse.com"));
+
+        List<String> actualList = slackTextHandler.getURLsFromText(text);
```

Comment:
I think variable name must not contain class name.
Actual and expected is common names. Or maybe 'urls'